src/test-routes/dpg/update.ts

```diff
@@ -1,17 +1,74 @@
 import { app, json, ValidationError } from "../../api";
 import { coverageService } from "../../services";
 
+/**
+ * Scenarios to test DPG and RLC Service driven evolution.
+ * These scenarios here are considered acceptable from an Azure breaking change policy
+ */
 app.category("dpg", () => {
+  /**
+   * Initially only has one required Query Parameter. After evolution, a new optional query parameter is added.
+   */
   app.get("/servicedriven/parameters", "DPGAddOptionalInput", (req) => {
+    if (req.query["parameter"]) {
+      coverageService.track("dpg", "DPGAddOptionalInput");
+      return {
+        status: 200,
+        body: json({ message: `An object was successfully returned` }),
+      };
+    } else {
+      return {
+        status: 400,
+        body: json({ message: `Expected required parameter "parameter"` }),
+      };
+    }
+  });
+
+  /**
+   * Initially has no query parameters. After evolution, a new optional query parameter is added
+   */
+  app.head("/servicedriven/parameters", "DPGAddOptionalInput_NoParams", (req) => {
+    coverageService.track("dpg", "DPGAddOptionalInput_NoParams");
+    return {
+      status: 200,
+      headers: { "content-length": "0" },
+    };
+  });
+
+  /**
+   * Initially has one required query parameter and one optional query parameter.  After evolution, a new optional query parameter is added
+   */
+  app.put("/servicedriven/parameters", "DPGAddOptionalInput_RequiredOptionalParam", (req) => {
+    if (req.query["requiredParam"]) {
+      coverageService.track("dpg", "DPGAddOptionalInput_RequiredOptionalParam");
+      return {
+        status: 200,
+        body: json({ message: `An object was successfully returned` }),
+      };
+    } else {
+      return {
+        status: 400,
+        body: json({ message: `Expected required parameter "requiredParam"` }),
+      };
+    }
+  });
+
+  /**
+   * Initially has one optional query parameter. After evolution, a new optional query parameter is added
+   */
+  app.get("/serviceDriven/moreParameters", "DPGAddOptionalInput_OptionalParam", (req) => {
+    coverageService.track("dpg", "DPGAddOptionalInput_OptionalParam");
```

Comment:
Shouldn't need to manually track if it is the same as the name above

Reply:
Oh cool thanks! I'll remove the explicit track